cosim/black-parrot-example/ps.cpp

```diff
@@ -28,6 +28,23 @@
 #define BP_NCPUS 1
 #endif
 
+// Offsets for GP0 (Check bsg_zynq_pl_shell in top_zynq)
```

Comment:
I like these changes a lot; can we pull them into a separate PR so that the diff is more clear about the watchdog vs these CSR changes?